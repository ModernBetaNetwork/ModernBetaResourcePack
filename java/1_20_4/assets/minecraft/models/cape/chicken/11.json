{
	"parent": "minecraft:cape/cape_11",
	"textures": {
		"0": "item/capes/chicken",
		"particle": "item/capes/chicken"
	}
}