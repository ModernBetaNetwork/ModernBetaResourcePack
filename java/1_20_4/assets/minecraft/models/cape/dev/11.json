{
	"parent": "minecraft:cape/cape_11",
	"textures": {
		"0": "item/capes/dev",
		"particle": "item/capes/dev"
	}
}