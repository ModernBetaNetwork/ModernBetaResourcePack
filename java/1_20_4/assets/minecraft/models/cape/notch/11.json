{
	"parent": "minecraft:cape/cape_11",
	"textures": {
		"0": "item/capes/notch",
		"particle": "item/capes/notch"
	}
}