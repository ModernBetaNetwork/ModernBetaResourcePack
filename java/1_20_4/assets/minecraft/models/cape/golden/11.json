{
	"parent": "minecraft:cape/cape_11",
	"textures": {
		"0": "item/capes/golden",
		"particle": "item/capes/golden"
	}
}